config.go

```diff
@@ -468,12 +468,9 @@ func loadConfig() (*config, []string, error) {
 		os.Exit(0)
 	}
 
-	// Exit if you tried to do rollback testing on a network other than
-	// simnet.
-	if cfg.RollbackTest && !cfg.SimNet {
-		fmt.Fprintln(os.Stderr, "Tried to do rollback testing of "+
-			"wallet for network other than simnet!")
-		os.Exit(0)
+	// Warn if rollback testing is enabled, as this feature was removed.
```

Comment:
I'd almost prefer dieing on old config (since people ignore warnings) but I'm fine with it as is.